charts/policy-reporter/templates/role.yaml

```diff
@@ -1,13 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
the ConfigMap Role is required to load and watch the priority config map configuration

Reply:
got it, maybe it makes sense to mount this CM into pod and then check? Also, i don't know why that's required additional `namespace`. Could it be in the same namespace?

Reply:
then i'll add optional enable or disable because i don't use that priority and it creates additional `role` which isn't so good in security perspective

Reply:
Make it optional is fine and the role is in the same namespace, also the role binding namespace is set with the release namespace.

I don't mount it because I use the content in ne dynamic matter and the ConfigMap is optional and not part of the helm chart.

Reply:
ok, i've added 
```
policyPriorities:
  enabled: false
```

Reply:
Also, if needs i can add configmap, to create that via helm
if it's necessary let me know

Reply:
Okay, I will updated the documentation about it afterwards. The CM should be optional and not be managed by helm for now. Thank you